apps/console/src/features/claims/components/add/add-external-claim.tsx

```diff
@@ -21,10 +21,10 @@ import { AlertLevels, Claim, ClaimsGetParams, ExternalClaim, TestableComponentIn
 import { addAlert } from "@wso2is/core/store";
 import { Field, FormValue, Forms, Validation, useTrigger } from "@wso2is/forms";
 import { PrimaryButton } from "@wso2is/react-components";
-import React, { FunctionComponent, ReactElement, useEffect, useState } from "react";
+import React, { FunctionComponent, ReactElement, useEffect, useState, SyntheticEvent } from "react";
 import { useTranslation } from "react-i18next";
 import { useDispatch } from "react-redux";
-import { Grid, Header } from "semantic-ui-react";
+import {Grid, Header, DropdownOnSearchChangeData, StrictDropdownItemProps, DropdownItemProps} from "semantic-ui-react";
```

Comment:
Can we keep the space after the curly brace?
```suggestion
import { Grid, Header, DropdownOnSearchChangeData, StrictDropdownItemProps, DropdownItemProps } from "semantic-ui-react";
```